src/pattern.rs

```diff
@@ -93,6 +93,7 @@ lazy_static! {
     pub static ref EPISODE: Pattern = regex!(
         r"(?i)e(?P<short>\d+)(?:[^\d]|$)|(episode)(?:[^\d]|$)(?P<long>\d+)|\d+x(?P<cross>\d+)|s\d+ - (?P<dash>\d+)"
     );
+    pub static ref FILE_EXTENSION: Pattern = regex!(r"(?i)(?:\.)(?P<extension>[a-z]{2,3}(?:\d)?)$");
```

Comment:
That's a cool trick, and it works surprisingly well from the samples I'm getting, but I'd like to also have support for `.h264` and `.webm`

Reply:
I have never seen for first 2 as file extensions. But I can take a look and see if I can add them. I neglected `m4v`

If you have a list. I can see what I can do.

Reply:
Those should be enough. They're fairly rare as most people use Mpeg or Matroska container but I've definitely seen some torrent with all of those.

Try to keep it in the same style though, new or esoteric video containers extension will probably always fall in that 2-4 alphanumeric category.

Reply:
Something like this looks ok. It's not as clean as I would like

```
(?:\.)(?P<extension>[a-z]{2,4}(?:\d)?|m4v|3gp|h26[45])$
```

Note that this is only intended to match `files` 

such as 
```
something.avi
something.m4v
``` 

and will not match the existing examples and should NOT match. The `.` is part of the limiting condition.

Reply:
Tested on []RustExp(https://rustexp.lpil.uk
<img width="1003" alt="scapshot" src="https://user-images.githubusercontent.com/6175103/151339333-216f22a9-2656-4786-b524-4fd2d9a09812.png">
)

Let me know what you think an will get it in sometime tomorrow.